src/ui/components/AppWrapper/AppWrapper.tsx

```diff
@@ -464,6 +466,13 @@ const AppWrapper = (props: { children: ReactNode }) => {
         }
       }
     }
+    const pendingDeletions =
```

Comment:
Actually, this needs to run when we successfully connect to KERIA. Right now, it'll try to do this even if we hit the catch clause a few lines up, that won't work.

And it's needs to be robust, for example if we had 5 connections to delete:
- Connected
- Deleted 2
- KERIA crashed
- Re-connect
- Should delete the other 3